src/App.js

```diff
@@ -32,13 +32,13 @@ function App() {
     notificationData.push(notificationItem);
   }
 
-  return (
-    <NotificationsPanel class="NotificationsFlyout"
+  return notificationData.length ?
+  <NotificationsPanel class="NotificationsFlyout"
       heading="Notifications"
       indicatorTitle="View application alerts"
       notifications={notificationData}>
-    </NotificationsPanel>
-  );
+  </NotificationsPanel>
+  : null;
```

Comment:
But if we do have 0 news, is the behavior still expected?

Reply:
Hi, in this case, you were right. If the data was empty no messages would be displayed.
To update the data that we are grabbing from the API and display it when it's ready, I've set a default value (false) to the APIData variable. When the request is finished, this variable is fulfilled and the data will refresh the display values.
https://reactjs.org/docs/hooks-state.html